src/index.ts

```diff
@@ -265,7 +266,8 @@ async function postJSON<T>(config: Config, url: string | URL, body = {}): Promis
       'Content-Type': 'application/json',
       'User-Agent': `database-js/${Version}`,
       Authorization: `Basic ${auth}`
-    }
+    },
+    cache: 'no-store'
```

Comment:
This broke all Cloudflare deployments 😅

cc @dgraham

![image](https://user-images.githubusercontent.com/599459/236140057-01988fbe-8313-4be4-b1ee-fc46a3d9c9d6.png)

Reply:
https://github.com/cloudflare/workers-sdk/issues/3005